### PR TITLE
[16.0] [IMP] web_refresher: Add component to pager to avoid instance for all views

### PR DESCRIPTION
**Pull request by CarlosRoca13**
Forward-port with adaptation of #2243 

cc @Tecnativa TT30229

please review this @chienandalu @pedrobaeza

### CODE REVIEW

File: web_refresher/static/src/js/pager.esm.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/** @odoo-module **/
+/* Copyright 2022 Tecnativa - Carlos Roca
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+import {Pager} from "@web/core/pager/pager";
+import {Refresher} from "./refresher.esm";
+
+// Patch control panel to initialize refresher component
```
Comment (chienandalu) [inline web_refresher/static/src/js/pager.esm.js]: This comment can go away I think :slightly_smiling_face: Or at least rephrase it with something like: we add the refresher component to the Pager, that is hooked into every control panel.

**Review by chienandalu — APPROVED** [review]
Great work! Aside from a tiny comment :+1: OWL Rules!

**Comment by CarlosRoca13** [comment]
Comment removed @chienandalu

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Pull request merged**